examples/with-cookie-auth/server.js

```diff
@@ -0,0 +1,78 @@
+const axios = require('axios')
+const express = require('express')
+const cookieParser = require('cookie-parser')
+const next = require('next')
+
+const dev = process.env.NODE_ENV !== 'production'
+
+const AUTHENTICATED_TYPE = 'authenticated'
+const COOKIE_SECRET = 'some deep dark secret'
+const COOKIE_OPTIONS = {
+  // domain: 'YOU_DOMAIN',
+  path: '/',
+  secure: !dev,
+  httpOnly: true,
+  signed: true
+}
+
+// fake serverless authentication
+const authenticate = async (email, password) => {
+  const users = await axios.get('https://jsonplaceholder.typicode.com/users').then(response => response.data)
```

Comment:
Using https cause error in development. Better use something like `const scheme = dev ? 'http' : 'https'`?